test/libsolidity/syntaxTests/nameAndTypeResolution/421_interface_function_parameters.sol

```diff
@@ -2,4 +2,4 @@ interface I {
     function f(uint a) public returns (bool);
 }
 // ----
-// Warning: (18-59): Functions in interfaces should be declared external.
+// TypeError: (18-59): Functions in interfaces must be declared external.
```

Comment:
Same here: I suspect this test is meant to pass, so it should be changed to ``external``.